image.go

```diff
@@ -20,10 +20,12 @@ type Image interface {
 	Target() ocispec.Descriptor
 	// Unpack unpacks the image's content into a snapshot
 	Unpack(context.Context, string) error
-	// RootFS returns the image digests
+	// RootFS returns the unpacked diffids that make up images rootfs.
 	RootFS(ctx context.Context) ([]digest.Digest, error)
-	// Size returns the image size
+	// Size returns the total size of the image's packed resources.
```

Comment:
@stevvooe is this correct? size is the (potentially compressed) layer size prior to unpacking?

Reply:
I copied this from https://github.com/containerd/containerd/blob/master/images/image.go#L58. :)

Reply:
Ha.. well then that makes sense :) Actually looking at the function those are blobs with OCI/Docker media types, so it must be the downloaded content prior to unpack, which could definitely be compressed.

Reply:
This only walks the resources in manifests.